guidelines/sondaggi.md

```diff
@@ -0,0 +1,38 @@
+# Sondaggi
+
+Il seguente documento è stato redatto per fornire una guida al team Ambassador per la creazione di sondaggi e le regole che vi stanno attorno.
+
+## Creazione di un sondaggio
+
+Per richiedere la creazione di un sondaggio è sufficiente fornire al Communication Group le seguenti informazioni:
+
+- Motivo del sondaggio
+- Titolo del sondaggio
+- Lista di opzioni
+- Risposta multipla consentita (Sì/No)
+- Sondaggio anonimo (Sì/No)
+
+Attualmente non vengono stabilite delle linee guida sulla valenza dei sondaggi, ma si consiglia di utilizzare questo strumento solo per questioni di rilevanza generale.
+
+Il Communication Group si impegna a rispondere entro 48 ore dalla richiesta di creazione di un sondaggio.
+
+## Tempi di risposta
+
+I tempi di risposta per i sondaggi sono fissati a 3 giorni (72 ore) dalla data di creazione del sondaggio.
+
+Questo tempo è da ritenersi sufficiente qualora venga raggiunto il quorum minimo. In caso contrario, il sondaggio verrà prorogato di ulteriori 4 giorni.
+
+## Quorum
+
+Il quorum minimo per la validità di un sondaggio è fissato al 20% (Arrotondato per difetto) del totale dei membri del team Ambassador.  
+Es. 20% di 10 membri = 2 membri, 20% di 41 membri = 8 membri.
+
+Qualora siano presenti motivate assenze, la persona assente verrà considerata "membro votante" a votazione nulla, concorrendo al quorum totale. Questo per evitare che un sondaggio venga invalidato per l'eventuale eccessiva concomitanza di assenze.
+
+## Validità del sondaggio
+
+Un sondaggio è da considerarsi valido se:
+
+- Il quorum minimo è stato raggiunto
+- Il sondaggio ha ricevuto un numero di voti pari o superiore al 50% del quorum minimo qualora si trattasse di una situazione _Sì/No_.
+- Qualora il sondaggio contenga più di due opzioni disponibili, l'opzione con più voti che superi il 33% dei votanti verrà considerata valida.
```

Comment:
Non mi piace molto. Significa che se fossimo 30 votanti e ci fossero 6 opzioni, per essere valido almeno 10 persone dovrebbero votare una singola opzione. Non è un pò limitante nel caso in cui ci fossero tante opzioni?

Reply:
Beh, oddio, arrivare a 10 la vedo difficile, parliamo di situazione in cui 3 è già un numero sufficiente in quanto solitamente si discute PRIMA di arrivare al sondaggio.. In ogni caso se su 10 opzioni meno del 33% convergono su una significa che non c'è un reale consenso, non trovi? Forse in quel caso è meglio risedersi al tavolo e riparlarne.

Come la gireresti tu?

Reply:
Mi vengono in mente queste cose:

1. Limitare il numero di opzioni ammissibili e ridurle a un tetto massimo prima di creare il sondaggio. Questo approccio richiede di stabilire un limite massimo al numero di opzioni che possono essere incluse nel sondaggio prima di avviare il processo di votazione. In caso di eccesso di opzioni, è necessario ridurle a questo limite massimo e quindi si re-itera la discussione. Tuttavia, può risultare burocratico e ostacolare la decisione se non viene raggiunto un quorum minimo di opzioni per il sondaggio
2. La scelta con il maggior numero di opzioni vince, indipendentemente da tutto. Questo approccio è molto semplice e diretto. La decisione finale è presa in base al numero di opzioni a favore di ciascuna scelta. Tuttavia, è molto semplicistico e potrebbe non riflettere completamente le preferenze dei partecipanti, dando il vantaggio a una scelta con una maggioranza parziale ma non necessariamente alla preferenza più ampiamente accettata

Reply:
Proporrei un meccanismo di ballottaggio - se ci sono tante opzioni e non si raggiunge un consenso (51%), si ripete il sondaggio con le sole due che hanno preso più voti.

Reply:
Può andare il ballottaggio, ma a quel punto ridurrei a 24h il secondo voto per non dilungare troppo.